weapon-config.inc

```diff
@@ -1042,7 +1043,19 @@ stock ReturnWeaponName(weaponid)
 
 	return name;
 }
-
+stock EnableHealthBarForPlayer(playerid, bool:enable)
+{
+	if (playerid != INVALID_PLAYER_ID)
```

Comment:
Please instead use `IsPlayerConnected` and not a check against `INVALID_PLAYER_ID` because -1 would get through this count and possibly cause an out of bounds error

Reply:
Hm, okay i will change that.